INSTALL.md

```diff
@@ -34,91 +35,41 @@ apt-get update
 
 ### Installation
 ```bash
-wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/mysterium-node_linux_amd64.deb
-sudo dpkg --install --force-depends mysterium-node_linux_amd64.deb
+wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/myst_linux_amd64.deb
+sudo dpkg --install --force-depends myst_linux_amd64.deb
 sudo apt-get install -y --fix-broken
 ```
+
 ### Running service
 ```bash
 sudo service mysterium-node start
```

Comment:
`service myst-node start` or `service myst start`

Reply:
Maybe we can have service aliases and support both names:
- mysterium_node
- myst -> mysterium_node

Reply:
But I imagine future like this:
- myst <- name of binary
- `mysterium_node` service which starts `myst service ..` service
- `mysterium_client` service which starts `myst connect ..` service

Reply:
1) I don't think that we need to add aliases, it will be just extra confusion for the users. We will need to have extra documentation to describe the reasons for aliases. I do not see any advantages having two names for the single service. 

2) In the current implementation, it works exactly the way you described. `myst` is a bynary, `mysterium_node` service that starts `myst service ...`

Reply:
Cool